docs/expo.md

```diff
@@ -0,0 +1,163 @@
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.Signal;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.consent.Consent;
+```
+
+```kotlin
+class MainApplication : Application(), ReactApplication {
+  ...
+  override fun onCreate() {
+    super.onCreate()
+    ...
+
+    MobileCore.setApplication(this)
+    MobileCore.setLogLevel(LoggingMode.VERBOSE)
+    MobileCore.configureWithAppID("YOUR-APP-ID")
+    val extensions: List<Class<out Extension?>> = Arrays.asList(
+      Lifecycle.EXTENSION,
+      Signal.EXTENSION,
+      Edge.EXTENSION,
+      Consent.EXTENSION,
+      com.adobe.marketing.mobile.Identity.EXTENSION
+    )
+    MobileCore.registerExtensions(extensions,
+      AdobeCallback { o: Any? ->
+        MobileCore.lifecycleStart(
+          null
+        )
+      })
+  }
+}
+```
+
+> To enable the Lifecycle metrics, [implement the Lifecycle APIs](./packages/core/README.md#lifecycle)
+
+# Guide for Bare React Native apps using Expo modules
+
+## Overview
+Bare React Native workflows can be integrated with Expo SDKs by using the `install-expo-modules` command. This allows you to use Expo modules in your app.
+- To use Expo modules in your app, you will need to install and configure the expo package.
+- The expo package has a small footprint; it includes only a minimal set of packages that are needed in nearly every app and the module and autolinking infrastructure that other Expo SDK packages are built with.
+- Once the expo package is installed and configured in your project, you can use `npx expo install` to add any other Expo module from the SDK.
+
+## Installation
+- To install and use Expo modules, the easiest way to get up and running is with the `install-expo-modules` command.
+```bash
+npx install-expo-modules@latest
+```
+- If the command fails, please follow the manual installation [instructions](https://docs.expo.dev/bare/installing-expo-modules/#manual-installation).
+
+## Install Adobe Mobile SDKs
+
+Follow the [Installation Guide](../README.md#Installation) to install Adobe SDKs.
+
+## Initialize Adobe Mobile SDKs
+
+Follow the [Initialization Guide](../README.md#initializing) to initialize Adobe SDKs.
+
+# Troubleshooting and Known Issues
+1. Getting error when building on iOS
+```xcode
+error: import of C++ module 'Foundation' appears within extern "C" language linkage specification [-Wmodule-import-in-extern-c]
+```
+**Fix**: In XCode, add `-Wno-module-import-in-extern-c` to `Apple CLang - Custom Compiler Flags -> Other C++`
+<img width="936" alt="Screenshot 2024-08-23 at 1 15 03 AM" src="https://github.com/user-attachments/assets/518b0e39-a2dd-4f37-94c2-c7663cb4edcd">
+
+#### Other Known Issues with React Native
+Please refer [here](../README.md#troubleshooting-and-known-issues) for other known issues with React Native integration.
```

Comment:
## Other Known Issues with React Native

Refer to [Troubleshooting and known issues](../README.md#troubleshooting-and-known-issues) for other known issues with React Native integration.